snap/snapcraft.yaml

```diff
@@ -1,55 +1,74 @@
 name: srsran
 base: core20
-version: '22.04.1'
+version: '20.10'
 summary: srsRAN is a 4G/5G software radio suite developed by SRS.
 description: srsRAN is a 4G/5G software radio suite developed by SRS.
-
 grade: devel
 confinement: devmode
-
 apps:
+  srsenb:
+    command: usr/bin/srsenb
+    plugs:
```

Comment:
Are the plugs working? I.e. is the snap working when running in non classic/devmode?

Reply:
For now the snap doesn't work in confinement modes other than `devmode` for example it doesn't work with strict confinement, this is because it won't be able to create `seqpacket` socket.
For now there is no interface in snapcraft that allows this if I'm not mistaken.
However these interfaces are needed too for this snap to work in strict confinement.
I think it's good to keep them even if the list is not yet complete and the snap can only run in `devmode` for now.
Or do you think it would be better to remove them?

Reply:
I think we should remove the plugs as they make it seem like the snap would work in strict confinement. We can keep track of them in an issue or elsewhere but I don't think they should remain in the snap if they are unused. Aiming for a classic snap is a good enough start.

Reply:
Okay, removed :)